### DISCUSSION TITLE
update for jdk 17

### DISCUSSION THREAD

**Discussion by kaeufler** [microsoft/openjdk#213] [Q&A] [answered]
Where is the update for jdk 17 after latest news about vulnerable in java jdk

**Comment by karianna** [comment]
Hi there,

Which vulnerability / CVE are you referring to?



On Mon, 24 Jan 2022 at 13:27, kaeufler ***@***.***> wrote:

> Where is the update for jdk 17 after latest news about vulnerable in java
> jdk
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/openjdk/discussions/213>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABME2GCNFM6SDNXP6YMBIDUXVHVHANCNFSM5MVMAY3Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Cheers, Martijn (Sent from Gmail Mobile)

**Comment by kaeufler** [comment]
The last notified in german security message.

Mit freundlichem Gruß
gez. W. Käufler
diese Nachricht ist elektronisch signiert.
Wilfried Käufler
Obere Bahnhofstraße 12			
34376 Immenhausen			
***@***.***
Telefon: 05673/912154
Fax:05673/9290328
Mobil:0176/20078885

Am 24.01.2022 um 17:54 schrieb Martijn Verburg:
> Hi there,
>
> Which vulnerability / CVE are you referring to?
>
>
>

**Comment by karianna** [comment]
Hi there,

Apologies but I'm not sure which one that would be referring to exactly?
You can report the details to ***@***.*** and we'll take
a look.

Cheers,
Martijn

**Reply by KalleOlaviNiemitalo** [reply]
As a bystander, I see a bunch of asterisks in that email address. Does GitHub show the real address to the original poster?

**Reply by karianna** [reply]
Hm, yeah GitHub swallowed that.  Hmm let me think of a better way

**Reply by KalleOlaviNiemitalo** [reply]
There's an email address shown at <https://github.com/microsoft/openjdk/blob/main/SECURITY.md#reporting-security-issues>; is that the right one for this?

(I have no idea what the vulnerability is.)

**Reply by karianna** [reply]
That is correct.  FYI - there are no CVE's of High+ rating in the latest Java quarterly CPU (coming out this week), so I suspect this is a Log4shell warning, but let's see :-)

**Comment by karianna** [comment, marked as answer]
OK, I've reached out to the German Authorities on this to see what it is they sent out about Java.  If it's related to the MS Build of OpenJDK, I'll reply here, otherwise consider it unrelated.

**Reply by karianna** [reply]
OK, it's unrelated.  Existing CVEs related to Java 17 are all < 6 in CVSS score and will be resolved in next week's 17.0.2 release regardless
